### DISCUSSION TITLE
Any way to make image-overlays clickable as links?

### DISCUSSION THREAD

**Discussion by blackpeasy** [javalent/obsidian-leaflet#277] [Q&A]
I use leaflet in combination with excalidraw to map out a story I am writing. I want to use it as a dashboard of my story, so I can click on any of the text elements, taking me to a note of the character, the location etc.

Is there any way for me to make these text bits clickable as links? 
- Map markers looks very big and messy in this context. Any way to make 'text'-markers?
- I thought about making transparent overlays over the text and then making the overlay a clickable link. Is that possible?

<img width="400" alt="image" src="https://user-images.githubusercontent.com/92858268/151825027-befa0858-6402-4324-820d-516e9008497d.png">

**Comment by Servinjesus1** [comment]
This would actually be really awesome, because it opens the idea up for true [image maps](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/map), one of the [coolest](https://imagemappro.com/) most underused knowledge management tools that blow mind maps out of the water. Ol' [DokuWiki](https://www.dokuwiki.org/plugin:imagemap) has a plugin for them, but finding them elsewhere is a bit of a rarity.

**Reply by blackpeasy** [reply]
Indeed. That is exactly my dream.
